docs/platforms/javascript/common/opentelemetry/custom-setup.mdx

```diff
@@ -0,0 +1,93 @@
+  dsn: "__DSN__",
+  skipOpenTelemetrySetup: true,
+
+  // The SentrySampler will use this to determine which traces to sample
+  tracesSampleRate: 1.0,
+});
+
+// Note: This could be BasicTracerProvider or any other provider depending on
+// how you are using the OpenTelemetry SDK
+const provider = new NodeTracerProvider({
+  // Ensure the correct subset of traces is sent to Sentry
+  // This also ensures trace propagation works as expected
+  sampler: sentryClient ? new SentrySampler(sentryClient) : undefined,
+});
+
+// Ensure spans are correctly linked & sent to Sentry
+provider.addSpanProcessor(new SentrySpanProcessor());
+
+provider.register({
+  // Ensure trace propagation works
+  // This relies on the SentrySampler for correct propagation
+  propagator: new SentryPropagator(),
+  // Ensure context & request isolation are correctly managed
+  contextManager: new Sentry.SentryContextManager(),
+});
+
+// Validate that the setup is correct
+Sentry.validateOpenTelemetrySetup();
+```
+
+Make sure that all <PlatformLink to="/opentelemetry/required-instrumentation/">Required OpenTelemetry Instrumentation</PlatformLink> is set up correctly - otherwise, the Sentry SDK may not work as expected!
+
+## Using Sentry for Error Monitoring only
+
+If you have a custom OpenTelemetry setup and only want to use Sentry for error monitoring, you can skip adding the `SentrySpanProcessor`. You will still need to add the `SentryContextManager`, `SentryPropagator` and `SentrySampler` to your setup even if you do not want to sent any tracing data to Sentry. See below on an explanation of why this is needed.
+
+## Why do I need to do this?
+
+In order for the Sentry SDK to work as expected, and for it to be able to be in sync with OpenTelemetry, we need a few components to be in place. These are needed even if you only use Sentry for error monitoring:
+
+- **SentryContextManager**: This is needed to ensure that the OpenTelemetry context is in sync with Sentry.
+- **SentrySampler**: This is needed to ensure that the Sentry `tracesSampleRate` is respected. Even if you do not use Sentry for tracing, you still need this in order for trace propagation to work as expected. See <PlatformLink to='/opentelemetry/sampler'>Using a Custom OpenTelemetry Sampler</PlatformLink> if you want to use a custom sampler.
+- **SentryPropagator**: This is needed to ensure that trace propagation works correctly.
+- <PlatformLink to="/opentelemetry/required-instrumentation">Required Instrumentation</PlatformLink>: These are needed to ensure that trace propagation works correctly.
+
+<Note>
+  Trace propagation is needed for Sentry to automatically connect services (e.g.
+  frontend and backend, or different backend services) together. This makes it
+  possible to see related errors across services.
+  <PlatformLink to="/tracing/trace-propagation">Learn more about Trace Propagation.</PlatformLink>
+</Note>
+
+The following is only needed if you use Sentry for tracing:
```

Comment:
To confirm, `SentrySpanProcessor` is needed in addition to everything else if Sentry is also used for tracing, right? I'd point this out more clearly:

```suggestion
In addition, the following is only needed if you also use Sentry for tracing:
```